cli/crates/backend/src/api/graphql/types/mutations/environment_variable_delete.rs

```diff
@@ -47,7 +47,9 @@ pub struct EnvironmentVariableDeleteByValuesSuccess {
 
 #[derive(cynic::InlineFragments, Debug)]
 pub enum EnvironmentVariableDeleteByValuesPayload {
+    #[allow(dead_code)]
     EnvironmentVariableDeleteByValuesSuccess(EnvironmentVariableDeleteByValuesSuccess),
+    #[allow(dead_code)]
```

Comment:
These seem like they _should_ be used, no?  Or if we don't need them, should they be removed?

Reply:
Yeah let's delete all dead code